### ISSUE TITLE
Payments Block Formatting

### ISSUE THREAD

**Issue by sbathompson-he** [Automattic/jetpack#38790]
### Impacted plugin

Jetpack

### Quick summary

The Payment Block does not allow the width full settings to be applied. This could be a theme issue related to Assembler, but in looking at the CSS, it looks to be tied to a CSS that is `customize-unpreviewable`. Changing this also changes things like Social Icons, which doesn't seem right. 
<img width="1696" alt="Screenshot 2024-08-08 at 9 18 37 AM" src="https://github.com/user-attachments/assets/8916f905-2aeb-49e6-befd-c70a60c2bdec">
<img width="1699" alt="Screenshot 2024-08-08 at 9 18 19 AM" src="https://github.com/user-attachments/assets/80cbb420-7a49-44e7-b664-48ab340c26eb">

https://github.com/user-attachments/assets/e34d07a9-3a70-47d1-9513-5fa7dd8aeea3

### Steps to reproduce

1. Add Payment Block to site
2. Set width to 100%
3. Preview or View, not at 100%

### A clear and concise description of what you expected to happen.

It should resolve to any width a user sets.

### Impact

Some (< 50%)

### Available workarounds?

No and the platform is unusable

**Comment by sbathompson-he** [comment]
Reported here: 8543698-zd-a8c

**Comment by mpkelly** [comment]
I've reproduced this now on Simple with T24, Hevor, and Celcius. Going to investigate further and then try to fix it. 

<img width="1074" alt="Screenshot 2567-08-21 at 13 18 16" src="https://github.com/user-attachments/assets/e87c7331-6b8a-4303-bd0b-1e3af5014b90">

<img width="872" alt="Screenshot 2567-08-21 at 13 18 19" src="https://github.com/user-attachments/assets/13b8b130-73fd-42d4-9b67-f075593d302a">

**Comment by mpkelly** [comment]
It feels like a broken experience all around. The editor and front end never match. I can't get two buttons to remain on the same row in the editor even though I've changed the settings:

<img width="1060" alt="Screenshot 2567-08-21 at 13 37 11" src="https://github.com/user-attachments/assets/c09dec33-22e2-474a-9c99-48a7d6943663">

<img width="717" alt="Screenshot 2567-08-21 at 13 37 22" src="https://github.com/user-attachments/assets/62a129b7-f967-4435-a92b-611f4da8fa47">

I can work around this issue by putting a single button in each payment buttons block and putting them in a columns block, but that defeats the purpose of the payment buttons block. Plus, the width issue remains. 

I am going to look at fixing this now.

**Comment by mpkelly** [comment]
I've made some progress with this and have identified two issues:

1. Editor: the contain is set to `block` which should probably be `flex`. I pinged the author of this file for a review
2. Front-end: the [styles in this file](https://github.com/Automattic/jetpack/blob/34fa923b37061eb143ba74fde55ab3544b7e739f/projects/plugins/jetpack/extensions/blocks/payment-buttons/view.scss) are not being loaded. If I patch them manually, the front-end issues are fixed. I have asked on #jetpack-developers for some guidance debugging this one as I'm not familiar with how Jetpack is packaged on WPCOM.
